auto_tune_vllm/core/study_controller.py

```
@@ -1061,11 +1092,33 @@ def _run_baseline_trials(self):
             if concurrency > 256:
                 baseline_parameters["max_num_seqs"] = concurrency
 
+            # Enqueue baseline trial in Optuna study with fixed parameters
+            # This ensures it appears in the dashboard and visualizations
+            try:
+                self.study.enqueue_trial(baseline_parameters)
+                logger.debug(
+                    f"Enqueued baseline trial with parameters: {baseline_parameters}"
+                )
+            except Exception as e:
+                logger.error(f"Failed to enqueue baseline trial: {e}")
+                continue  # Skip to next concurrency level - prevents study corruption
```

> @aas008 Where are the baseline's parameter values being included/set in the study object ?

@thameem-abbas this is where.

---

This can confuse the sampler with empty data. As I understand this weakens the statistical model. I'm not sure when the idea changed to include it as trial 0 instead of just adding the information as a study user attribute. 

Can we potentially figure out a way to make use the baseline log parsing you already had going ?

---

If we can enforce at least 10 or 15 startup trials, I think we might be okay here but would need some digging in here. @ephoris Does my hypothesis here make sense ?

---

I think this would depend on the sampler a lot. From what I read about GP and TPE is that they learns that "Trial with unknown parameters got xyz tokens/sec" &  cannot use this data point to build the model. But I haven't verified this for all samplers we could use, so I understand that this approach will not be sustainable then, I will work on populating all fields.
Thanks @thameem-abbas!

---

@aas008
We currently have it user configurable with the default set to None/0. I think we should fail and warn the user about it rather than setting the default to 10. 

I think the quickest solution with a reasonable effort in play is to note this down clearly in the code and add an assert for the n_startup_trials less than 10. Can allow an override with an env var if we want.